client/app/components/SortIcon.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular';
+
+export function SortIcon({ column, sortColumn, reverse }) {
```

Comment:
FYI, once tables are converted to Ant `<Table>`, the "Filter and sorter" feature would render this component obsolete.

Reply:
But until that time we need this component